delivery_postlogistics/wizard/stock_picking_return.py

```diff
@@ -0,0 +1,11 @@
+from odoo import models
+
+class ReturnPicking(models.TransientModel):
+    _inherit = "stock.return.picking"
+    
+    def _create_returns(self):
```

Comment:
this will impact all returns
can you add a condition `on self.picking_id.delivery_type == "postlogistics"`